### DISCUSSION TITLE
Ranking results

### DISCUSSION THREAD

**Discussion by abdullah2993** [community/community#21018] [Other Feature Feedback, Questions, & Ideas]
It would be really nice to be able to rank the search results, two of the most important metrics while ranking should be repo stars and when was the repo last updated or overall activity on the repository.
This might not be very important for regular searching but when you are searching for exploring new stuff or find some very specific implementation/usage of something it would be really nice to be able to see the results from most active repos. Right now you might end up getting results from repos which are either not being maintained or are personal projects.

**Comment by jfullstackdev** [comment]
is this the one you are referring too ? 

![image](https://user-images.githubusercontent.com/47092464/180434997-4646c0b8-83b3-4ae3-b089-bfe295141473.png)

**Reply by abdullah2993** [reply]
> is this the one you are referring too ?
> 
> ![image](https://user-images.githubusercontent.com/47092464/180434997-4646c0b8-83b3-4ae3-b089-bfe295141473.png)

in code search not repository search

**Comment by colinwm** [comment]
Are you using cs.github.com yet? We are working on ranking (based on a number of factors, including stars) there.

**Reply by jsoref** [reply]
Yes, this is a question about cs.

Also, I'd like to be able to sort by the number of direct forks a repository has. (And probably by the total number of forks a repository has, but generally, my default would be direct forks.) The reason for favoring direct over indirect is that if there was a repository and its owner abandoned it, and then one of its forks became more popular and people started making forks off that direct fork of the original repository, I'd want to see that fork of the original and not the original fork.
